cmd/fleet/main.go

```diff
@@ -471,19 +471,24 @@ LOOP:
 
 		// Create or recreate cache
 		if configCacheChanged(curCfg, newCfg) {
+			log.Info().Msg("reconfigure cache on configuration change")
 			cacheCfg := makeCacheConfig(newCfg)
 			err := f.cache.Reconfigure(cacheCfg)
-			log.Info().Err(err).Interface("cfg", cacheCfg).Msg("Reconfigure cache")
+			log.Info().Err(err).Interface("cfg", cacheCfg).Msg("reconfigure cache complete")
```

Comment:
nit: here the log message was changed to start with lower case, but the handleCheckin.go change in the same PR it's upper case